main.go

```diff
@@ -683,7 +687,7 @@ func main() {
 
 	http.Handle("/sets", enforceSigning(http.HandlerFunc(Search)))
 	http.Handle("/sealed", enforceSigning(http.HandlerFunc(Search)))
-
+	http.
```

Comment:
http?

Reply:
this was where i was going to add the handler but then ?? elmo happened